src/main/java/duke/TextStoreAndPrint/TextPrinter.java

```diff
@@ -0,0 +1,72 @@
+package duke.TextStoreAndPrint;
+
+public class TextPrinter {
+
+    public static void standardPrint(String middle) {
```

Comment:
Might be able to standardize with the rest of your method

Reply:
you mean like naming scheme wise? like standardising with `commandCheck(String string)` ?